content/docs/getting-started/adding-components/index.md

```diff
@@ -64,6 +64,18 @@ Click the Home link in the Backstage sidebar to go back to the catalog where you
 
 ![our component visible on the other tab of Backstage](./component-on-other-tab.png)
 
+## Configuring Backstage to auto discover github repositories using the github discovery processor
+
+The Roadie team can configure your backstage instance to auto discover catalog yaml files created in your github repositories. To do this, please contact one of us and we will be happy to configure it for you.
```

Comment:
```suggestion
The Roadie team can configure your Backstage instance to auto discover `catalog-info.yaml` files created in your GitHub repositories. To do this, please contact us via Slack and we will be happy to configure it for you.
```

Fixed some capitalization issues. `catalog-info.yaml` is the canonical name of the files so you can use that exact term.